db/migration/20130108000009_create_users.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[4.2]
+  def change
+    if table_exists? :users
+      change_table :users do |t|
+        t.string  :password_digest,  limit: 60
```

Comment:
As authentication migrate to ActiveRecord's `has_secure_password`, we need to add password_digest column to users. Without this, existing users cannot update their password.